contract-map.json

```diff
@@ -225,6 +225,13 @@
     "decimals": 18
   },
   "0x8762db106B2c2A0bccB3A80d1Ed41273552616E8": {
+    "name": "Reserve Rights (DO NOT USE)",
+    "logo": "rsr.svg",
+    "erc20": true,
+    "symbol": "RSR",
```

Comment:
Unfortunately symbol in this collection can not overlap

Reply:
thanks, I will remove the old token record